README.md

```diff
@@ -57,7 +59,7 @@ You can install real extension and use automatically created client in local con
 This extension allows end users to login, not dashboard administrators. This means that we need to secure this extension in the same way that we secure other applications in Auth0.
 
  1. Create a "Single Page Application" in Clients
- 2. Put `http://localhost:3001/login` as an `Allowed Callback URL`.
+ 2. Put `http://localhost:3000/login` as an `Allowed Callback URL`.
```

Comment:
You can use 3000, but webpack is listening to 3001. So if you want to work with extension's UI, you should use 3001.